python/guide/getting-started/to-mxnet/pytorch.md

```diff
@@ -25,15 +18,18 @@ For MXNet we use pip:
 # !pip install mxnet
 ```
 
-## Multidimensional matrix
+Both PyTorch and Apache MXNet have separate packages if you want to use CUDA. To install Apache MXNet with GPU support, you need to specify CUDA version. For example the snippet below will install Apache MXNet with CUDA 9.2 support:
```

Comment:
sugegest removing the first sentence: ~~Both Pytorch and Apache MXNet have separate packages ...~~